### PR TITLE
support for 8/16 output pin config

### PR DESCRIPTION
**Pull request by moinmoin-sh**
This version adds support for 8/16 pin configurations for output ports.
It's fully copatible to the latest API updates and ready for use with the Admin-UI-WEB-APP.
Requires the updated LCD-lib.

### CODE REVIEW

**Comment by moinmoin-sh** [comment]
I had to move the initialisation of the IO-Handlers into the scanI2Cbus() function to get these set before the config from the persistant SPIFFS takes place. If they were initialised in configI2Cbus() after Rack32.begin() the restored config was overwritten.

File: OXRS-SHA-StateIO-ESP32-FW.ino [hunk 1]
```diff
@@ -182,13 +209,21 @@ void setConfigSchema()
   
   JsonObject ioConfig = json.createNestedObject("ioConfig");
   ioConfig["type"] = "string";
+  ioConfig["description"] = "Hint ! Restart your device when changed to make effective!";
```
Comment (sumnerboy12) [inline OXRS-SHA-StateIO-ESP32-FW.ino]: Can I suggest changing these to "A restart is required before changes will take effect".

**Review by sumnerboy12 — APPROVED** [review]
Looks good, just the suggestion for the hint text.

File: OXRS-SHA-StateIO-ESP32-FW.ino (hunk 1 above)
Comment (moinmoin-sh) [reply OXRS-SHA-StateIO-ESP32-FW.ino]: Thank's for the sugestion. It was my expectation that you would come up with a better wording.

**Pull request merged**